src/EFCore.PG/Migrations/NpgsqlMigrationsSqlGenerator.cs

```diff
@@ -64,7 +64,7 @@ public class NpgsqlMigrationsSqlGenerator : MigrationsSqlGenerator
                                 NpgsqlValueGenerationStrategy.SerialColumn => true,
                                 _ => false
                             })
-                            .Select(p => p.GetColumnName()))
+                            .Select(p => p.GetColumnName(StoreObjectIdentifier.Table(t.Table, t.Schema))))
```

Comment:
Why are there no local?

Reply:
Local variable for the result of StoreObjectIdentifier.Table? There's no real impact, StoreObjectIdentifier is just a struct anyway, and there's no perf impact (otherwise I'd have never done this big LINQ)